solidity/dashboard/src/components/LiquidityRewardCard.jsx

```diff
@@ -23,13 +26,31 @@ const LiquidityRewardCard = ({
   lpBalance,
   isFetching,
 }) => {
+  const dispatch = useDispatch()
+
   const formattedPercentageOfTotalPool = useMemo(() => {
     const bn = new BigNumber(percentageOfTotalPool)
     return bn.isLessThan(0.01) && bn.isGreaterThan(0)
       ? "<0.01"
       : bn.decimalPlaces(2, BigNumber.ROUND_DOWN)
   }, [percentageOfTotalPool])
 
+  const addLpTokens = async (awaitingPromise) => {
```

Comment:
I think we should pass submit functions as props and the parent(container) component should specify how to handle these buttons. I believe this gives us more flexibility, e.g. when we would add another pair but with a different API. Also, the `LiquidityRewardCard` component will be independent of redux.